### ISSUE TITLE
Nodes API error sending and receiving messages in emqx 5.0.10

### ISSUE THREAD

**Issue by siniusxcode** [emqx/emqx#9353]
### What happened?

<img width="1726" alt="스크린샷 2022-11-11 오후 5 32 24" src="https://user-images.githubusercontent.com/1418182/201300622-813c382e-2767-42cf-8519-0f4f9fb45662.png">

Increasing the number of messages sent and received results in nodes API delays and errors.
I tested it as below using [emqtt-bench](https://github.com/emqx/emqtt-bench).

./emqtt_bench sub -c 2000 -i 1 -h {host} -p 1883 -k 0 -t bench/%i -q 2
./emqtt_bench pub -c 2000 -I 10 -h {host} -p 1883 -k 0 -t bench/%i -s 1024

As clients, subscriptions, and messages increase in version 5.0.10, the related APIs(clients, subscriptions, nodes) seem to suffer from poor performance.
This did not happen with my emqx 4.0.7 cluster.

### What did you expect to happen?

I hope that overall API performance will be improved and there will be no problem with the dashboard.

### EMQX version

<details>

```console
$ ./bin/emqx_ctl broker
sysdescr  : EMQX
version   : 5.0.10
datetime  : 2022-11-11T17:50:14.310923503+09:00
uptime    : 1 hours, 39 minutes, 11 seconds
```

</details>

### OS version

<details>

```console
# On Linux:
$ cat /etc/os-release
# paste output here
$ uname -a
# paste output here

# On Windows:
C:\> wmic os get Caption, Version, BuildNumber, OSArchitecture
# paste output here
```

</details>

### Log files

<details>

</details>

**Comment by zhongwencool** [comment]
Plan fix in v5.0.16

**Comment by qzhuyan** [comment]
In the fix of #9856 we added a workaround with caching. 
Looks like your system is overloaded. You might need to scale up your EMQX node. 
You might hit other issue if you don't scale up the nodes.
